### DISCUSSION TITLE
Change DNS

### DISCUSSION THREAD

**Discussion by tswiderski1** [Security-Onion-Solutions/securityonion#9621] [Unsupported Versions]
How to change DNS in Security Onion standalone? I changed in /etc/resolv.conf but observes that my old DNS is still being queried by SO.

**Comment by cm-ops** [comment]
You could modify with NetworkManager, `nmcli con mod $connection ipv4.dns "xxx.xxx.xxx.xxx,xxx.xxx.xxx.xxx"`. Then restart the service `systemctl restart NetworkManager`.
